tools/generate_comparison_plots.py

```diff
@@ -1,3 +1,10 @@
+import os.path
+import sys
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
```

Comment:
Hi there, guys! 
My only comment here is that in the ROSS code base we are trying to have the pathlib library as a standard for taking care of paths. 

Here the code would be something like:

```python
import sys

from pathlib import Path

import numpy as np
import plotly.io as pio
from jinja2 import Environment, FileSystemLoader

sys.path.append(Path(__file__).parent.parent)
import ross as rs
```

Reply:
We'll fix this right away @raphaeltimbo!